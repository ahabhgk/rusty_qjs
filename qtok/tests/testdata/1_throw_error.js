function main() {
  throw new Error("ee");
}

main();
