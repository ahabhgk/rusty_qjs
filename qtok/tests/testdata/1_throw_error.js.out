[exception] Error: ee
    at main ([WILDCARD]/tests/testdata/1_throw_error.js:2)
    at <anonymous> ([WILDCARD]/tests/testdata/1_throw_error.js:5)

